apps/etterlatte-tilbakekreving/src/main/kotlin/Application.kt

```diff
@@ -26,8 +28,12 @@ class Server(
             applicationConfig = ConfigFactory.load(),
             withMetrics = false,
         ) {
-            testKravgrunnlagRoutes(context.kravgrunnlagService)
             tilbakekrevingRoutes(context.tilbakekrevingService)
+
+            if (context.properties.enableKravgrunnlagRoutes) {
+                application.log.warn("Legger på endepunkter for kravgrunnlag (kun tilgjengelig lokalt)")
```

Comment:
Kunne evt. ha kastet feil hvis vi kjører i gcp / prod her?

Reply:
Gjorde en endring her. Kaster nå feil ved gcp, og unngår å konsumere MQ ved lokal kjøring da dette ikke støttes for m*-mac'er.